wremnants/HDF5Writer.py

```diff
@@ -190,12 +190,14 @@ def write(self,
             hist_nominal = dg.groups[procs_chan[0]].hists[chanInfo.nominalName] 
             hist_axes[chan] = [hist_nominal.axes[a] for a in axes]
 
+            hist_nominal_perProc = {}
             # nominal predictions
             for proc in procs_chan:
                 logger.debug(f"Now  in channel {chan} at process {proc}")
-                
+
                 # nominal histograms of prediction
                 norm_proc_hist = dg.groups[proc].hists[chanInfo.nominalName]
+                #hist_nominal_perProc[proc] = norm_proc_hist.copy()
```

Comment:
This line and the L193 where "hist_nominal_perProc" is defined can now be deleted again

Reply:
Done